astroplan/data/observatories.json

```diff
@@ -381,4 +381,18 @@
         "name": "mro",
         "source": "MRO website"
     }
-}
\ No newline at end of file
+        "SALT" : {
```

Comment:
This is picky, but could you correct the indentation on this line and include a source for this information? It's ok to say "SALT website" as your source.